examples/Azure_IoT_Adu_ESP32/readme.md

```diff
@@ -63,7 +63,23 @@ _The following was run on Windows 11 and Ubuntu Desktop 20.04 environments, with
   - ESP32 boards are not natively supported by Arduino IDE, so you need to add them manually.
   - Follow the [instructions](https://github.com/espressif/arduino-esp32) in the official ESP32 repository.
 
-## Setup Instructions
+
+## Add Azure IoT Hub Device to an ADU Deployment Group
```

Comment:
It won't let me comment at the top, but the table of contents at the top of the file isn't in the right order anymore

Reply:
Thanks, good catch!

Reply:
Fixed.